specificatie/genereervariant/openapi.json

```diff
@@ -2675,7 +2953,7 @@
         }
       }
     },
-    "/kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}/publiekrechtelijkebeperkingen/{publiekrechtelijkebeperkingidentificatie}" : {
+    "/publiekrechtelijkebeperkingen/{publiekrechtelijkebeperkingidentificatie}" : {
       "get" : {
         "tags" : [ "Publiekrechtelijke Beperkingen" ],
         "description" : "Het raadplegen van een specifieke publiekrechtelijke beperkingen op een kadastraal onroerende zaak.\n\nMet gebruik van de parameter expand kunnen gerelateerde stukdelen direct worden meegeladen.\n",
```

Comment:
\n\nMet gebruik van de parameter expand kunnen gerelateerde stukdelen direct worden meegeladen.\n",

moet worden: door deze parameter te gebruiken kun je gerelateerde stukdelen direct meeladen.

Reply:
Dit is niet de description bij de expand parameter, maar van het endpoint. Dan moet je toch zeggen welke "deze parameter" is?

Deze zinsopbouw hebben we overal in de BRK API gebruikt (in release 1.0.0). Bijvoorbeeld bij /kadastraalonroerendezaken staat "Met gebruik van de parameter expand kunnen zakelijkgerechtigden direct worden meegeladen."